go/node/node.go

```diff
@@ -0,0 +1,7 @@
+package node
+
+type Node interface {
```

Comment:
I don't think this interface is needed, is it used anywhere?

Reply:
It's a good idea to make sure that Node implementations follow it. 
I've changed the noderunner to use the interface and added a todo to have the testrunner run the docker instance tests.

On a side note, we're slowing starting to build a good suite of long-running tests that we should probably run every night?

Reply:
Fair enough. Not sure what you mean about the long-running tests, I'll ping on discord.